xarm_planner/src/xarm_simple_planner.cpp

```diff
@@ -85,7 +85,7 @@ void XArmSimplePlanner::init()
   exec_plan_srv = node_handle.advertiseService("xarm_exec_plan", &XArmSimplePlanner::exec_plan_cb, this);
 
   visual_tools = new moveit_visual_tools::MoveItVisualTools("link_base");
-  Eigen::Affine3d text_pose = Eigen::Affine3d::Identity();
+  Eigen::Isometry3d text_pose = Eigen::Isometry3d::Identity();
```

Comment:
I guessed it is because we are using newer Eigen3?

Reply:
Yes, this causes an error in the version of Eigen3 we are using.